ci-operator/step-registry/openstack/provision/bastionproxy/openstack-provision-bastionproxy-commands.sh

```diff
@@ -132,21 +138,40 @@ http_access deny !Safe_ports
 http_access deny CONNECT !SSL_ports
 http_access allow localnet
 http_access deny all
-http_port 8213
+http_port 3128
+https_port 3130 cert=/etc/squid/certs/domain.crt key=/etc/squid/certs/domain.key cafile=/etc/squid/certs/domain.crt
+# Leave coredumps in the first cache dir
+coredump_dir /var/spool/squid
+auth_param basic program /usr/lib64/squid/basic_ncsa_auth /etc/squid/passwd
+auth_param basic children 5
+auth_param basic realm Squid Basic Authentication
+auth_param basic credentialsttl 2 hours
+acl auth_users proxy_auth REQUIRED
+http_access allow auth_users
 EOF"
+
+sudo mkdir -p /etc/squid/certs
+cd /etc/squid/certs
+sudo openssl req -newkey rsa:4096 -nodes -sha256 -keyout domain.key -x509 -days 365 \
```

Comment:
nit: make it 1 day, in case the server is staled, nobody can use the proxy > day.
We can do this later.

Reply:
done via https://github.com/openshift/release/pull/21307